sdk/dslink/src/main/java/org/dsa/iot/dslink/provider/netty/DefaultWsProvider.java

```diff
@@ -132,9 +132,7 @@ public void write(EncodingFormat format,
                         byte[] bytes = data.encode(format);
                         ByteBuf buf = Unpooled.wrappedBuffer(bytes);
                         WebSocketFrame frame = null;
-                        if (format == EncodingFormat.MESSAGE_PACK) {
```

Comment:
so I was reading this yesterday... we don't want to use msgpack anymore? Or were we even using it at all?

Reply:
We weren't using it. It's dead code that depends on a vulnerable library.